pom.xml

```diff
@@ -274,7 +278,7 @@
             <id>default-test</id>
             <configuration>
               <excludesFile>src/test/resources/slow-or-flaky-tests.txt</excludesFile>
-              <argLine>${surefire.argLine}</argLine>
+              <argLine>@{jacoco.surefire.argLine} ${surefire.argLine}</argLine>
```

Comment:
To explain what happened: we are setting our own arg line now and, if set, the JaCoCo agent doesn't inject its options.

The change injects again the JaCoCo options.